spec/acceptance/timeout_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+
+describe "Timeout" do
+  context "raise Almodovar::HttpClient::TimeoutError exception" do
```

Comment:
The exception class is actually just `Almodovar::TimeoutError`. I'd amend this just to avoid confusion.